be/src/runtime/fragment_mgr.cpp

```diff
@@ -653,12 +653,26 @@ Status FragmentMgr::_get_or_create_query_ctx(const TPipelineFragmentParams& para
             }
 
             if (!query_ctx) {
+                WorkloadGroupPtr workload_group_ptr = nullptr;
+                std::string wg_info_str = "not carry wg";
+                if (params.__isset.workload_groups && !params.workload_groups.empty()) {
+                    uint64_t wg_id = params.workload_groups[0].id;
+                    workload_group_ptr =
+                            _exec_env->workload_group_mgr()->get_task_group_by_id(wg_id);
```

Comment:
rename  this method get_task_group_by_id to get_group  
not using task group
not add id